lib/java_buildpack/container/tomcat/tomcat_geode_store.rb

```diff
@@ -44,8 +44,10 @@ def release
         credentials = @application.services.find_service(FILTER, KEY_LOCATORS, KEY_USERS)['credentials']
         user = credentials[KEY_USERS].find { |u| cluster_operator?(u) }
 
-        @droplet.java_opts.add_system_property 'gemfire.security-username', user['username']
-        @droplet.java_opts.add_system_property 'gemfire.security-password', user['password']
+        File.open(@droplet.root + 'gemfire.properties', 'w') do |f|
+          f.write('security-username=' + user['username'] + "\nsecurity-password=" + user['password'])
+        end
+
         @droplet.java_opts.add_system_property 'gemfire.security-client-auth-init',
```

Comment:
We could. I'm not sure if there's any benefit to seeing that method name printed out in the `JAVA_OPTS`, but I'll discuss it with Ben.

Reply:
Ben and I discussed it, we think it might be useful to leave this in the JAVA_OPTS in case anyone needs it for debugging purposes.  Also, I guess it makes the most sense now that we've removed the `gemfire.properties` file.